test/ooni/templates.cpp

```diff
@@ -173,6 +175,73 @@ TEST_CASE("http requests template works as expected") {
     });
 }
 
-#else
-int main(){}
 #endif
+
+static void mocked_request(Settings settings, http::Headers,
+        std::string, Callback<Error, SharedPtr<http::Response>> cb,
+        SharedPtr<Reactor>, SharedPtr<Logger>,
+        SharedPtr<http::Response>, int) {
+    std::string probe_ip = settings.get("real_probe_ip_", std::string{});
+    REQUIRE(probe_ip != "");
```

Comment:
If we ever change the way in which we pass around the probe_ip, this will break.